python/metatensor-operations/tests/sort.py

```diff
@@ -282,6 +323,10 @@ def test_sort_descending(tensor, tensor_sorted_descending):
     )
 
 
+def test_sort_empty_block(tensor_with_empty_block, tensor_sorted_ascending):
+    metatensor.sort_block(tensor_with_empty_block.block(2))
```

Comment:
Can you add some assert checking that the block is properly sorted?

Reply:
sure, I am on it

Reply:
ok, i basically wrote a test that all other blocks are still ordered properly. But thinking longer about it, it does not really make sense since this is all block-wise anyways.. The empty block is not sorted of course, so I cannot assess the sorting

Reply:
this is what I had in mind! This makes sure that the presence of an empty block does not change the behavior for other blocks!